packages/ssz/src/type/optional.ts

```diff
@@ -55,24 +59,34 @@ export class OptionalType<ElementType extends Type<unknown>> extends CompositeTy
     return null as ValueOfType<ElementType>;
   }
 
-  getView(tree: Tree): ValueOfType<ElementType> {
-    return this.tree_toValue(tree.rootNode);
+  // TODO fix
+  // eslint-disable-next-line @typescript-eslint/no-unused-vars
+  getView(_tree: Tree): ValueOfType<ElementType> {
+    throw new Error("not implemented");
```

Comment:
@wemeetagain should we not depending on its composite type or not provide the toValue or underlying view?